### DISCUSSION TITLE
How do I get `bun run` to not truncate the output of my script/command?

### DISCUSSION THREAD

**Discussion by dyc3** [oven-sh/bun#15058] [Q&A] [answered]
When I run scripts, I generally need to be able to see the full output of the script to be able to read error messages. Instead, bun truncates the output and prints out `[X lines elided]` and just the last 10 ish lines.

How do I get bun to print the full output like a normal program?

**Comment by hubdidub** [comment]
I think it's your script that truncates the output, not Bun. Bun doesn't truncate the output of my scripts.

**Reply by dyc3** [reply]
It's definitely not my script. When I was using `yarn` the output wasn't truncated. I think it might have something to do with using `--filter`.

**Reply by dyc3** [reply]
Actually, I found an issue that describes my issue exactly: #12431

**Reply by hubdidub** [reply]
ah ok, yes that's definitely it.
In that case, I'd like to point out this comment from the issue you linked: https://github.com/oven-sh/bun/issues/12431#issuecomment-2332717606. Using `concurrently` might be a solution to your problem.

**Comment by RiskyMH** [comment, marked as answer]
`--elide-lines=<num>` feature added by @martinamps in https://github.com/oven-sh/bun/pull/15837 and released in Bun v1.1.43!

You can use `0` to disable completely.
